app/src/views/features/rules/RuleEditor/components/Header/ActionButtons/CreateRuleButton/actions/insertScriptValidators.js

```diff
@@ -94,7 +94,11 @@ export const HTML_ERRORS = {
  */
 export async function validateHTMLTag(str, htmlNodeName) {
   const HTMLLinterResult = await htmlValidateRawCodeString(str);
-  if (!HTMLLinterResult.isValid) {
+
+  if (
+    !HTMLLinterResult.isValid &&
+    !HTMLLinterResult.validationErrors.find((ele) => ele.errorType === "unclosed_tags")
+  ) {
```

Comment:
Shouldn't this be OR condition?

Reply:
No because we want to avoid this particular HTML linting error as this will get fixed by the changes here